templates/quickstart-jira-dc-with-vpc.template.yaml

```diff
@@ -567,7 +567,7 @@ Parameters:
     Type: String
   KeyPairName:
     ConstraintDescription: Must be the name of an existing EC2 Key Pair.
-    Description: Public/private EC2 Key Pairs to allow you to securely access the Bastion host
+    Description: Public/private EC2 Key Pairs(without file extension) to allow you to securely access the Bastion host.
```

Comment:
You're also missing a space between `Pairs` and the first open bracket `(`

Reply:
Done!